client/src/components/MainPage/MainContents/Log/style.ts

```diff
@@ -123,15 +123,17 @@ export const TagTitle = styled.div`
   border-radius: 0.25rem;
 `;
 
-export const TaskItems = styled.div`
+export const TaskItems = styled.div<{
```

Comment:
해당 변수는 단일 아이템을 나타내는 것 같은데 제가 잘못 이해한 걸까요?
```suggestion
export const TaskItem = styled.div<{
```

Reply:
https://github.com/boostcampwm-2022/web19-Boostart/pull/49/commits/7143a39574da57f6e15e164c5c36a6dfa79ea74e

해당 커밋에 수정되었습니다.